numpy_challenge.py

```diff
@@ -0,0 +1,53 @@
+# 3 ways to create an array
+if __name__ == "__main__":
+    print(np.arange(7))
+    print(np.ones(7))
+    print(np.eye(7))
+    
+# 1. This function takes 2 matrices and multiplies them (NOT by elements) and returns the result
+def matrix_multiplication(m, n):
+    return np.dot(m, n)
```

Comment:
Неинформативный нейминг, не понятно, что за `m` и `n`. Чуть более интуитивно было бы `a` и `b`, ещё лучше `matrix1` и `matrix2`
```suggestion
def matrix_multiplication(matrix1, matrix2):
    return np.dot(matrix1, matrix2)
```